### PR TITLE
Animation placeholder import

### PR DESCRIPTION
**Pull request by Dejin1**
Added only 1 script most of it is animations and art placeholders

### CODE REVIEW

File: Assets/Editor/GroupCommand.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+using UnityEditor;
+using UnityEngine;
+
+public static class GroupCommand
+{
+    [MenuItem("GameObject/Group Selected %g")]
+    private static void GroupSelected()
+    {
+        if (!Selection.activeTransform) return;
```
Comment (massieb) [inline Assets/Editor/GroupCommand.cs]: Surround with curly braces {} to prevent future bugs

File: Assets/Editor/GroupCommand.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
+using UnityEditor;
+using UnityEngine;
+
+public static class GroupCommand
```
Comment (massieb) [inline Assets/Editor/GroupCommand.cs]: Explain what this does above the class

File: Assets/Scripts/GameControlScript.cs [hunk 3]
```diff
@@ -43,29 +43,29 @@ void LateUpdate()
 
        
 
-            switch (Player1.GetComponent<Health>()._HP)
-        {
-            case 3:
-                //Full health
+        //    switch (Player1.GetComponent<Health>()._HP)
```
Comment (massieb) [inline Assets/Scripts/GameControlScript.cs]: Commented code. Remove or implement the correct way

File: Assets/Scripts/Player/JinPlayerMovement.cs [hunk 4]
```diff
@@ -77,7 +77,7 @@ void Update()
 
             horizontalMove = Input.GetAxisRaw(stringHoizontal) * runspeed;
 
-            //animator.SetFloat("speed", Mathf.Abs(horizontalMove));//this has been disabled until animations come along
+            animator.SetFloat("speed", Mathf.Abs(horizontalMove));//this has been disabled until animations come along
```
Comment (massieb) [inline Assets/Scripts/Player/JinPlayerMovement.cs]: disabled comment can be removed

File: Assets/Scripts/Player/JinPlayerMovement.cs [hunk 5]
```diff
@@ -173,37 +173,36 @@ void Update()
...
         }//can Move
-
     }//end Update
 
 
 
     public void OnCrouching(bool isCrouching)
     {
-        //animator.SetBool("isCrouching", isCrouching); //Start Crounch Animation
+        //Debug.Log("crouch method called: " + isCrouching);
```
Comment (massieb) [inline Assets/Scripts/Player/JinPlayerMovement.cs]: commmented code can be removed or uncommented

File: Assets/Editor/GroupCommand.cs (hunk 1 above)
Comment (Dejin1) [reply Assets/Editor/GroupCommand.cs]: I dont even know what this script is or does I think I may delete it xD

File: Assets/Editor/GroupCommand.cs (hunk 2 above)
Comment (Dejin1) [reply Assets/Editor/GroupCommand.cs]: I have no idea I will have to find it and delete it

File: Assets/Scripts/GameControlScript.cs (hunk 3 above)
Comment (Dejin1) [reply Assets/Scripts/GameControlScript.cs]: Fixed in new commit

File: Assets/Scripts/Player/JinPlayerMovement.cs (hunk 5 above)
Comment (Dejin1) [reply Assets/Scripts/Player/JinPlayerMovement.cs]: Fixed in new commit

File: Assets/Editor/GroupCommand.cs (hunk 1 above)
Comment (Dejin1) [reply Assets/Editor/GroupCommand.cs]: deleted this in new commit

File: Assets/Editor/GroupCommand.cs (hunk 2 above)
Comment (Dejin1) [reply Assets/Editor/GroupCommand.cs]: idk so i deleted it lol

File: Assets/Scripts/Player/JinPlayerMovement.cs (hunk 4 above)
Comment (Dejin1) [reply Assets/Scripts/Player/JinPlayerMovement.cs]: Removed it in new commit

File: Assets/Scripts/Enemy/LeftRight.cs [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+    {
+        MoveHorizontal();
+    }
+    void MoveHorizontal()
+    {
+        var temp = GameObjectLeftRight.position;
+        //print(up);
+        if (left == true)
+        {
+            temp.x += leftSpeed;
```
[the remark this answers is not in the document]
Comment (Dejin1) [reply Assets/Scripts/Enemy/LeftRight.cs]: I fixed it in the new commit with a flip method Not sure how to make a method for the rest of the code. I know you said to use a parameter but. I'm not sure how?

[the remark this answers is not in the document]
Comment (massieb) [reply Assets/Scripts/Enemy/LeftRight.cs]: What you did is already a big improvement, You could also send the `left` boolean in the flip method:
`
public void flip(bool isLeftFlip) {
    left = isLeftFlip;
}
`

[the remark this answers is not in the document]
Comment (Dejin1) [reply Assets/Scripts/Enemy/LeftRight.cs]: Fixed in new commit thanks Marcel!

File: Assets/Scripts/Player/JinPlayerMovement.cs [hunk 7]
```diff
@@ -191,6 +192,16 @@ void Update()
                 crouch = false;
                 Debug.Log(" crouch false: ");
             }
+
+
+            if (Input.GetButtonDown(shootProjectile))
+            {
+                //if (CharacterAnimator.GetBool("isCrouching") == false)
```
Comment (massieb) [inline Assets/Scripts/Player/JinPlayerMovement.cs]: Make it a habit to never commit commented code. There are exceptions. But then there should be a TODO above the commented code part. Now nobody knows why this is commented and what it should do.

Comment (Dejin1) [reply Assets/Scripts/Player/JinPlayerMovement.cs]: I was going to solve it next commit but yeah I understand will fix that now

Comment (Dejin1) [reply Assets/Scripts/Player/JinPlayerMovement.cs]: finished

**Pull request merged**